src/reuse/modules/mobile/device.ts

```diff
@@ -0,0 +1,116 @@
+"use strict";
+
+import { VerboseLoggerFactory } from "../../helper/verboseLogger";
+import ErrorHandler from "../../helper/errorHandler";
+
+/**
+ * @class device
+ * @memberof Mobile
+ */
+export class Device {
+  private vlf = new VerboseLoggerFactory("mobile", "device");
+  private ErrorHandler = new ErrorHandler();
+
+  /**
+   * @function isAppInstalled
+   * @memberOf mobile.device
+   * @description Check wether given package/bundle app is installed or not in the device.
+   * @param {string} packageIdorBundleId - Android package Id, or iOS bundle Id.
+   * @returns {boolean} Returns true if specified app package/bundled installed in the device, or false.
+   * @example await mobile.device.isAppInstalled("com.google.android.apps.maps");
```

Comment:
@Templada : Previous comment in #287 
so this is checking for google maps to be installed?

Reply:
No, its an example, in general every Android/iOS app's can be recognised with its package name(Android) or bundle id(iOS).

Reply:
yeah, i meant that example if that was the package name of maps?